test/binCases/watch/info-verbosity-off/info-verbosity-off.test.js

```diff
@@ -22,12 +50,39 @@ test("info-verbosity-off", async(done) => {
 		"--info-verbosity",
 		"none"
 	]);
-	const { stdout, stderr } = result;
-	expect(stdout).toEqual(expect.anything());
-	expect(stdout).toContain("");
-	expect(stdout).not.toContain("webpack is watching the files…");
-
-	expect(stderr).toHaveLength(0);
-	expect(stdout).toMatchSnapshot();
-	done();
+
+	var outputCount = 0;
+	var hash1;
+
+	webpackProc.stdout.on("data", data => {
+		data = data.toString();
+
+		if (outputCount === 0) {
+			hash1 = extractHash(data);
+
+			const summary = extractSummary(data);
+
+			expect(summary).toEqual(expect.anything());
+			expect(summary).toContain("");
+			expect(summary).not.toContain("webpack is watching the files…");
+			expect(summary).toMatchSnapshot();
+
+			// change file
+			appendDataIfFileExists(__dirname, fileToChange, "//junk-comment");
+
+			outputCount++;
+		} else {
+			const hash2 = extractHash(data);
+
+			expect(hash2.hash).not.toBe(hash1.hash);
+			webpackProc.kill();
+			done();
+		}
+	});
+
+	webpackProc.stderr.on("data", error => {
+		// fail test case if there is any error
+		expect(true).toBe(false);
```

Comment:
I don't like this. Is there a way to avoid writing redundant test cases and throw an error instead of using expect?

Reply:
Calling `done` with something will throw an error :)

Reply:
Is there any way to avoid using a noop assertion for this?